.github/workflows/ci.yml

```diff
@@ -321,6 +324,7 @@ jobs:
       fail-fast: false
       matrix:
         runs-on: [ubuntu-latest, macos-14, windows-latest]
+        python-version: ["3.11", "3.13"]
```

Comment:
We shouldn't double up the docs, it's causing timeouts when checking URLs. Three's already pushing it sometimes.

Reply:
Yeah, just opened this to try and reproduce the related issue, feel free to rebase accordingly. Btw why not use `3.x` in those workflows?

Also it didn't seem to have worked, any notes on why it didn't use 3.13 there?

Reply:
IIRC the nox action always uses the same Python to run itself, regardless of what you ask for (either 3.11 or 3.12, don't remember). And then if you don't specify a Python, it uses the one it ran with.

Reply:
> The compose Fedora-41 is not available in the public Testing

Do you know what that failure is?

Reply:
> Do you know what that failure is?

Yeah, fedora branched a few days ago, and `testing-farm` didn't catch up yet. I've reported this morning and they should be fixing it by tomorrow.

> And then if you don't specify a Python, it uses the one it ran with.

I guess you mean if you don't specify the Python version inside the nox file or cli?

Reply:
Yes, exactly.